lisa/sut_orchestrator/azure/common.py

```diff
@@ -1241,9 +1251,16 @@ def load_environment(
         ) = get_primary_ip_addresses(
             platform, resource_group_name, vms_map[node_context.vm_name]
         )
+
+        connection_public_ip = node_context.public_ip_address
```

Comment:
Please submit a separated PR for private IP support.

Reply:
created https://github.com/microsoft/lisa/pull/2682 for private IP support